### PR TITLE
Fix mongodb_user ssl_cert_reqs param (#2963)

### PR DESCRIPTION
**Pull request by Lujeni**
##### ISSUE TYPE
- Bugfix Pull Request
##### SUMMARY

All details: https://github.com/ansible/ansible-modules-extras/issues/2963

### CODE REVIEW

File: database/misc/mongodb_user.py [hunk 1]
```diff
@@ -320,20 +320,24 @@ def main():
...
-                                 replicaset=replica_set, ssl=ssl,
-                                 ssl_cert_reqs=ssl_cert_reqs)
-        else:
-            client = MongoClient(login_host, int(login_port), ssl=ssl,
-                                 ssl_cert_reqs=ssl_cert_reqs)
+            params["replicaset"] = replica_set
+
+        if ssl:
+            params["ssl"] = ssl
+            params["ssl_cert_reqs"] = getattr(ssl_lib, module.params['ssl_cert_reqs'])
```
Comment (resmo) [inline database/misc/mongodb_user.py]: Shouldn't this be more like:

```
connection_params["ssl"] = ssl
connection_params["ssl_cert_reqs"] = getattr(ssl_lib, module.params['ssl_cert_reqs'])
```

?

**Comment by gregdek** [comment]
Thanks again to @Lujeni. This module is going into community review.

We notice that you are one of the maintainers (@lujeni) of this module, so when you think it's ready, please comment "shipit" and we will consider it for merging.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by resmo** [comment]
needs_info

**Comment by gregdek** [comment]
@Lujeni, a community member requested some information about this PR; for that reason, we put this PR into the 'needs_info' state.

When you have answered the questions, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

File: database/misc/mongodb_user.py (hunk 1 above)
Comment (Lujeni) [reply database/misc/mongodb_user.py]: @resmo yep i need :coffee: thanks (fixed)

**Comment by resmo** [comment]
LGTM

**Comment by gregdek** [comment]
Thanks again to @Lujeni for this PR. This PR was reviewed by an Ansible member. Marking for inclusion.

However, we would still appreciate if the maintainer(s) lujeni could look over it.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
Thanks @Lujeni for this PR. Unfortunately, it is not mergeable in its current state due to merge conflicts. Please rebase your PR. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

For help on how to do this cleanly please see http://docs.ansible.com/ansible/community.html#contributing-code-features-or-bugfixes

[This message brought to you by your friendly Ansibull-bot.]

**Comment by Lujeni** [comment]
`ready_for_review`

**Comment by gregdek** [comment]
Thanks @Lujeni. To the current maintainers, @lujeni please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by Lujeni** [comment]
`shipit`

**Comment by resmo** [comment]
Thanks!

btw, as the module as diverted between devel and stable-2.1, I will not backport this fix. However v2.2 is near.

**Pull request merged**